tpu-provisioner/README.md

```diff
@@ -8,20 +8,24 @@ The provisioning process starts with an unschedulable "leader" pod (pod with Job
 JobSet. Once the TPU slice is created, the remaining pods for each Job will be created and follow their leader pod
 onto the same slice it is running on.
 
-<img src="./docs/provisioning.excalidraw.png" width="50%"></img>
-
-Node Pools are cleaned up when the JobSet whose pods triggered the node pool creation is either completed, failed, or deleted.
+Node Pools are cleaned up when the JobSet whose pods triggered the node pool creation is either **completed, failed, or deleted**.
 
 ## Setup
 
-### Create a GKE Cluster with workload identity enabled
+### Create a GKE Cluster with workload identity enabled and no release channel
 
-The TPU Provisioner requires workload identity to be enabled. 
+The TPU Provisioner requires [Workload Identity Federation for GKE](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) to be enabled, and cannot be on a release channel (auto upgrades
+are disabled on node pools created by the TPU provisioner, to minimize disruptions to training workloads)
```

Comment:
There is not a period at the end of the sentence "The TPU Provisioner requires...".

Reply:
Woops, fixed.